internal/date/date.go

```diff
@@ -57,8 +57,8 @@ func FromTime(t time.Time) Date {
 
 	return Date{
 		year:  t.Format("2006"),
-		month: t.Format("1"),
-		day:   t.Format("2"),
```

Comment:
I've changed this to make tests a bit easier where we go from a `time.Time` to a `date.Date` as I kept getting failures for missing leading `0`s. If we can see problems with this going forward I'll find some other workaround but it felt like the least impactful.